src/GitHub.Exports/Exports/ExportMetadata.cs

```diff
@@ -42,4 +45,13 @@ public interface IViewModelMetadata
     {
         UIViewType ViewType { get; }
     }
+
+    public static class ExportViewAttributeExtensions
+    {
+        public static bool IsViewType(this UserControl c, UIViewType type)
+        {
+            Debug.Assert(c.GetType().GetCustomAttributesData().Any(x => x.AttributeType == typeof(ExportViewAttribute) && x.NamedArguments.Any() && x.NamedArguments[0].TypedValue.ArgumentType == typeof(UIViewType)), "Someone broke the ExportViewAttribute contract...");
```

Comment:
This assert doesn't actually assert the part that could be a null ref, namely x.NamedAttributes. I went ahead and added the proper assert in a commit.